### DISCUSSION TITLE
Support Print of browser

### DISCUSSION THREAD

**Discussion by SettingDust** [alangrainger/share-note#100] [Vote for upcoming features]
The content is empty now.
![图片](https://github.com/alangrainger/share-note/assets/19922286/67eeaf1e-f072-4475-87a9-fff99514d4a6)

**Comment by MicaelJarniac** [comment]
Am also really looking forward to this! I found this plugin when trying to print a note from Obsidian and failing. So I thought if I shared it as a web page, and then printed, that it'd work.

**Comment by huyz** [comment]
See https://github.com/alangrainger/share-note/issues/75
